src/plugins/workspace/public/components/workspace_form/utils.ts

```diff
@@ -423,24 +423,24 @@ export const generatePermissionSettingsState = (
   }
 
   const finalPermissionSettings = [...(permissionSettings ?? [])];
-  const userPermissionExists = finalPermissionSettings.find(
-    (setting) => setting.type === WorkspacePermissionItemType.User
-  );
-  const groupPermissionExists = finalPermissionSettings.find(
-    (setting) => setting.type === WorkspacePermissionItemType.Group
-  );
-  if (!userPermissionExists) {
-    finalPermissionSettings.push({
-      ...emptyUserPermission,
-      id: generateNextPermissionSettingsId(finalPermissionSettings),
-    } as typeof finalPermissionSettings[0]);
-  }
-  if (!groupPermissionExists) {
-    finalPermissionSettings.push({
-      ...emptyUserGroupPermission,
-      id: generateNextPermissionSettingsId(finalPermissionSettings),
-    } as typeof finalPermissionSettings[0]);
-  }
+  // const userPermissionExists = finalPermissionSettings.find(
+  //   (setting) => setting.type === WorkspacePermissionItemType.User
+  // );
+  // const groupPermissionExists = finalPermissionSettings.find(
+  //   (setting) => setting.type === WorkspacePermissionItemType.Group
+  // );
+  // if (!userPermissionExists) {
+  //   finalPermissionSettings.push({
+  //     ...emptyUserPermission,
+  //     id: generateNextPermissionSettingsId(finalPermissionSettings),
+  //   } as typeof finalPermissionSettings[0]);
+  // }
+  // if (!groupPermissionExists) {
+  //   finalPermissionSettings.push({
+  //     ...emptyUserGroupPermission,
+  //     id: generateNextPermissionSettingsId(finalPermissionSettings),
+  //   } as typeof finalPermissionSettings[0]);
+  // }
```

Comment:
```suggestion
```
Better to remove the code instead of comment out.

Reply:
Oh sorry, I missed this change. 😂